features/install.feature

```diff
@@ -37,9 +75,47 @@ Feature: Installation
         reporter: dot
 
       # Path patten used for finding tests (see https://github.com/isaacs/minimatch)
-      testFilePattern: '**/*_{spec,test}.{coffee,js}'
+      testFilePattern: '**/*_{spec,test}.coffee'
 
       # Files to include before all tests
       testHelpers:
         - spec/spec_helper.coffee
       """
+
+  Scenario: install with a .ls test helper
+    Given I am working on a Node.js project
+    When I run "mycha install" and enter "spec/spec_helper.ls"
+    Then my "package.json" now lists the devDependencies
+      | NPM MODULE |
+      | chai       |
+      | sinon      |
+      | sinon-chai |
+    And my project now has a file "spec/spec_helper.ls" containing
+      """
+      require! [chai, sinon]
+      chai.use require('sinon-chai')
+
+      global.chai = chai
+      global.expect = chai.expect
+      global.sinon = sinon
+
+      process.env.NODE_ENV = 'test'
+      """
+    And my project now has a file "mycha.yml" containing
+      """
+      # Environment variables to add to process.env when running mocha
+      mochaEnv: {}
+
+      # Default options to pass to mocha (can be overriden by command line options)
+      mochaOptions:
+        colors: true
```

Comment:
Okay. Sorry I only tested it with http://yaml-online-parser.appspot.com, where this works, and taken the idea from http://yaml.org/type/bool.html. Strange.